src/ethereum_spec_tools/evm_tools/t8n/__init__.py

```diff
@@ -328,15 +328,6 @@ def apply_body(self) -> None:
             requests_trie = self.fork.Trie(secured=False, default=None)
             requests_from_execution: Tuple[Bytes, ...] = ()
 
-            self.fork.set_storage(
-                self.alloc.state,
-                self.HISTORY_STORAGE_ADDRESS,
-                (
-                    (self.env.block_number - 1) % self.HISTORY_SERVE_WINDOW
-                ).to_be_bytes32(),
-                U256.from_be_bytes(self.env.parent_hash),
-            )
-
```

Comment:
This seems unrelated to the EIP in the PR title. Just want to make sure it is an intentional change.

Reply:
Good point. This is a remnant from the state of EIP-2935 and the state of the testing suite at the time the PR was raised. I will update this and check with EEST about whether we can continue to use the state tests for `evm_tools_tests`

Reply:
Updated to the latest test suite with support for the `state-test` flag